rbc/tests/test_omnisci_array_null.py

```diff
@@ -34,10 +34,11 @@ def test_array_null(omnisci, col):
     if col in ['i2', 'i8', 'f8']:
         omnisci.require_version((5, 7, 0),
                                 'Requires omniscidb-internal PR 5465 [rbc PR 330]',
-                                hash='4777a06b01')
-    # skipping bool test since NULL is converted to true - rbc issue #245
+                                level=1)
```

Comment:
instead of `level=1`, can we name those numbers? For instance:
* `OMNISCIDB_DEV_DOCKER=1`
* `OMNISCIDB_INTERNAL_PR=10`

Reply:
yes, labelling such additional requirements makes sense.

How about:
```
omnisci.require_version(version, message, label='release')
```
where `label` is in `['release', 'dev-docker', 'dev']` (that correspond to current levels 0, 1, and 10, respectively), and the environment may define
```
OMNISCIDB_DEV_LABEL=<label value>
```

Reply:
Actually, I am thinking that using the branch name for label makes more sense. When OMNISCIDB_DEV_LABEL and the specified label match, only then the particular test will be run.

Reply:
That's a better approach

Reply:
Fixed. See the doc-string of require_version. In short, when testing rbc against omniscidb master or any branch that is based on master, use
```
export OMNISCIDB_DEV_LABEL=master
```
When implementing a test that requires a specific branch of omniscidb, say, `foo`, then use
```
export OMNISCIDB_DEV_LABEL=foo
```
and in the test, use `omnisci.require_version(..., label='foo')`.